com.alovoa.alovoa-electron.yml

```diff
@@ -0,0 +1,78 @@
+app-id: com.alovoa.alovoa-electron
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
```

Comment:
This one needs a bump too.

Reply:
Thank, updated the base version to 22.08